app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ConfirmationsController < Devise::ConfirmationsController
+
+  private
+
+  def after_confirmation_path_for(resource_name, resource)
+  	byebug
```

Comment:
it's wrong

On Wed, May 20, 2015 at 4:24 PM, Chirag Aggarwal <chi6rag@gmail.com> wrote:

> wait, which pull request are you monitoring?
>
> On Wed, May 20, 2015 at 4:21 PM, B V Satyaram <notifications@github.com>
> wrote:
>
>> In app/controllers/confirmations_controller.rb
>> <https://github.com/chi6rag/HourGlass/pull/1#discussion_r30689647>:
>>
>> > @@ -0,0 +1,10 @@
>> > +class ConfirmationsController < Devise::ConfirmationsController
>> > +
>> > +  private
>> > +
>> > +  def after_confirmation_path_for(resource_name, resource)
>> > +  	byebug
>>
>> Plz make it a habbit to remove such code before commiting. Always view
>> diff before committing.
>> If it is necessary for you to commit such changes temporarily, please add
>> a comment like #TODO: Remove this before such code. The #TODO tag has
>> special meaning in most of the editors/IDEs
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/chi6rag/HourGlass/pull/1/files#r30689647>.
>>
>
>